kaggle_environments/main.py

```diff
@@ -62,12 +67,22 @@
     "--host", type=str, help="http-server Host (default=127.0.0.1)."
 )
 parser.add_argument(
-    "--out", type=str, help="Output file to write the results of the episode."
+    "--in", type=str, help="Episode replay file to load. Only works when the action is load."
```

Comment:
You can now render downloaded json with `kaggle-environments --in ./path/to/replay.json --display html --out replay.html`